.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,54 @@
+name: Chromatic Storybook deployment
+
+on:
+  push:
+    branches:
+      - "master"
```

Comment:
Did you intentionally omit the release branch?

Reply:
Yes. Active development happens on master, I never heard somebody uses release branch to check storybook

@ranquild do you think we need automatic storybook deployments at releases branches?

Reply:
No, master only OR when `chromatic` label is on